tests/img_info_t.py

```diff
@@ -461,6 +461,6 @@ def suite():
     test_suites = []
     test_suites.append(unittest.makeSuite(InfoUnit, 'test'))
     test_suites.append(unittest.makeSuite(InfoFunctional, 'test'))
-    test_suites.append(unittest.makeSuite(InfoCache, 'test'))
+    test_suites.append(unittest.makeSuite(TestInfoCache, 'test'))
```

Comment:
Are these lines even necessary anymore? Seems like this fix should result in more tests being run, but it looks like the same number as a previous run, so seems like py.test is just finding the tests without these lines. Do you think we can remove them?

Reply:
I don’t use them, I think pytest looks for subclasses of unittest.TestCase and functions/classes whose name includes “Test”. Yeah, I think we can ditch them.

Reply:
OK. Let me know if you want to remove any in this PR. Otherwise, I'll merge it and we can remove the lines in a future PR.

Reply:
I’ll go ahead and remove them now, it’s only a small extra change.